optuna/storages/rdb/storage.py

```diff
@@ -50,26 +50,29 @@ class RDBStorage(BaseStorage):
 
     Example:
 
-        We create an :class:`~optuna.storages.RDBStorage` instance with
-        customized ``pool_size`` and ``max_overflow`` settings.
-
-        .. code::
-
-            >>> import optuna
-            >>>
-            >>> def objective(trial):
-            >>>     ...
-            >>>
-            >>> storage = optuna.storages.RDBStorage(
-            >>>     url='postgresql://foo@localhost/db',
-            >>>     engine_kwargs={
-            >>>         'pool_size': 20,
-            >>>         'max_overflow': 0
-            >>>     }
-            >>> )
-            >>>
-            >>> study = optuna.create_study(storage=storage)
-            >>> study.optimize(objective)
+        Create an :class:`~optuna.storages.RDBStorage` instance with customized
+        ``pool_size`` and ``timeout`` settings.
+
+        .. testcode::
+
+            import optuna
+
+            def objective(trial):
+                x = trial.suggest_uniform('x"', -100, 100)
```

Comment:
[nits] why `x"`?

Reply:
Thanks for your comment!
This is just an easy mistake.
I fixed it.